src/OpenApi/test/Integration/snapshots/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=forms.verified.txt

```diff
@@ -86,32 +86,78 @@
             "multipart/form-data": {
               "schema": {
                 "type": "object",
-                "properties": {
-                  "todo": {
-                    "required": [
-                      "id",
-                      "title",
-                      "completed",
-                      "createdAt"
-                    ],
+                "allOf": [
+                  {
                     "type": "object",
                     "properties": {
-                      "id": {
-                        "type": "integer",
-                        "format": "int32"
-                      },
-                      "title": {
-                        "type": "string"
-                      },
-                      "completed": {
-                        "type": "boolean"
-                      },
-                      "createdAt": {
+                      "resume": {
                         "type": "string",
-                        "format": "date-time"
+                        "format": "binary"
+                      }
+                    }
+                  },
+                  {
+                    "type": "object",
+                    "properties": {
+                      "files": {
+                        "type": "array",
+                        "items": {
+                          "type": "string",
+                          "format": "binary"
+                        }
                       }
                     }
                   }
+                ]
+              },
+              "encoding": {
+                "multipart/form-data": {
+                  "style": "form",
+                  "explode": true
```

Comment:
I think you probably already explained this, but what does "explode" mean here?

Reply:
`explode` here means whether arrays/objects get exploded into separate arguments.  For example, an array with explode=true is represented as `id=3&id=4&id=5` and with explode=false is represented as `id=3,4,5`. The explosion reflects the default behavior of our model binding for array arguments.